docs/data-abstraction/syllabus-data-abstraction.md

```diff
@@ -219,11 +231,11 @@ either a class, laboratory, or programming session are encouraged to seek
 assistance from the course instructor and the student technical leaders.
 Students should, within the bounds of the Honor Code, ask and answer questions
 on the Discord server for our course; please request assistance from the
-instructor and student technical leaders first through Discord Slack before
+instructor and student technical leaders first through Discord before
 sending an email. Students who need the course instructor's assistance must
-schedule a meeting through his web site and come to the virtual meeting with all
+schedule a meeting through the instructor's office hours calendar and come to the meeting with all
 of the details needed to discuss their question. Students can find out the
-office hour schedule student technical leaders by viewing the list of [student
+office hour schedule for student technical leaders by viewing the list of [student
```

Comment:
Yes, good catch! It is important to fix this writing mistake and the other writing mistakes in this paragraph.